data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -337,6 +337,7 @@ then
 	copy_static_resources_for openstack
 	copy_static_resources_for ovirt
 	copy_static_resources_for vsphere
+	copy_static_resources_for nutanix
```

Comment:
can we pull this change in to it's own commit since its not related to the core terraform file addition?